website/pages/docs/quickstart.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: Quickstart
+---
+
+import DownloadSection from '../../components/mdx/_getting-started-download-section.mdx'
+
+# Quickstart
+
+Follow the steps below to start syncing data from CloudQuery [source plugins](./plugins/sources) to [destination plugins](./plugins/destinations).
+
+## Download CloudQuery CLI
+
+<DownloadSection/>
+
+## Configure Destination Plugin
+
+Destination plugins define where you will be syncing your data to (See: [available destinations](./plugins/destinations)).
+
+For example, let's configure the [PostgreSQL](https://github.com/cloudquery/cloudquery/tree/main/plugins/destination/postgresql) destination plugin.
+
+```yaml
+kind: destination
+spec:
+  ## Required. name of the plugin
+  name: "postgresql"
+
+  # Required. Must be a specific version starting with v, e.g. v1.2.3
+  version: "vX.Y.Z"
+
+  ## Optional. Default: "overwrite". Available: "overwrite", "append". Not all modes are supported by all plugins,
+  ## so make sure to check the plugin documentation for more details.
+  write_mode: "overwrite" # overwrite, overwrite-delete-stale, append
+
+  spec:
+    ## plugin-specific configuration for PostgreSQL:
+
+    ## Required. Connection string to your PostgreSQL instance
+    connection_string: "postgresql://postgres:pass@localhost:5432/postgres?sslmode=disable"
+```
+
+For a list of all available destination plugin spec options, see [Destination Spec Reference](/docs/reference/destination-spec).
+
+## Configure Source Plugin
+
+Source plugins define the APIs you want to sync from (See [available sources](./plugins/sources)).
+
+For example, let's configure the [AWS](https://github.com/cloudquery/cloudquery/tree/main/plugins/source/aws) source plugin.
+
+```yaml
+kind: source
+spec:
+  ## Required. name of the plugin to use
+  name: "aws" # required
+
+  # Required. Must be a specific version starting with v, e.g. v1.2.3
+  version: "vX.Y.Z"
+
+  ## Optional. Default: "github". Available: "local", "grpc"
+  # registry: github
+
+  ## Optional. Default: cloudquery/name
+  # path: cloudquery/aws
+
+  ## Optional. Default: ["*"] - all tables. We recommend to specify specific tables that you need to sync as this
+  ## will reduce the amount of data synced and improve performance.
+  # tables: ["*"]
+
+  ## Required. all destinations you want to sync data to.
```

Comment:
I think we should show the `skip_tables` and possibly even populate with a few resources that were highlighted here: https://github.com/cloudquery/cloudquery-issues/issues/447